### ISSUE TITLE
Polishing Tasks v2

### ISSUE THREAD

**Issue by HazAT** [getsentry/sentry-mobile#81]
- [x] Onboarding Screens: Welcome, Explain what the app does (1-2 slides), Login 
  - [**Issue**](https://github.com/getsentry/sentry-mobile/issues/85)
  - 3 Screens with Dummy text
- [x] The Empty Screen (when pull to refresh) shouldn't show a full-page spinner, instead, it should show the content with one card and all the loading indicators in there
- [x] Fix bounding box of scroll container so it's not cut off on the bottom in the middle of the device
  - [**Issue**](https://github.com/getsentry/sentry-mobile/issues/87)
- [x] Write Username/Email besides Sign Out button (below Sign Out)
  - [**Issue**](https://github.com/getsentry/sentry-mobile/issues/88)
- [x] What do the two line charts (Issues / Crashes), Query?
  - Issues are aggregated stats of all issues, crashes are aggregated stats of all unhandled issues
- [x] Add a bottom line to highlight the 0-line in the chart
  - [**Issue**](https://github.com/getsentry/sentry-mobile/issues/89)

We need the new API:
- [x] Top card should no longer be the sessions of the latest release, instead, we use the new API to query Session Count over the last 12h
- [x] Use new Session API to calculate Crashfree Users + Session + Diff to previous 12h independent from releases and just focused on projects
- [x] Change bottom two cards to Stability Scope and Apdex (remove icons) (Apdex: https://github.com/getsentry/sentry/blob/640bdcd5a311a0dd613d17a958ef98125680b672/src/sentry/static/sentry/app/views/projectDetail/projectScoreCards/projectApdexScoreCard.tsx)

**Comment by HazAT** [comment]
How to calculate Apdex: 
https://github.com/getsentry/sentry/blob/640bdcd5a311a0dd613d17a958ef98125680b672/[…]views/projectDetail/projectScoreCards/projectApdexScoreCard.tsx

Session data API:
https://sentry.io/api/0/organizations/sentry/sessions/?project=11276&statsPeriod=12h&interval=1h&field=sum(session)

**Comment by matejminar** [comment]
@denrase To calculate the stability score we need to make two requests.

To get the data needed for stability score for the last 12h period:
https://sentry.io/api/0/organizations/sentry/sessions/?project=11276&statsPeriod=12h&field=sum(session)&groupBy=session.status

To get the data needed for stability score for the previous period (so that we can calculate diff/trend):
https://sentry.io/api/0/organizations/sentry/sessions/?project=11276&statsPeriodStart=24h&statsPeriodEnd=12h&field=sum(session)&groupBy=session.status

For each of those requests to get the stability score we need to calculate the percentage of crash free sessions:
100 - (crashed / (healthy + abnormal + crashed + errored)) * 100

You don't need to calculate with individual intervals - so in order to pick healthy sessions from the server response you would do something like this:
`data.groups.find(group => group.by['session.status'] === 'healthy').totals['sum(session)']`
same for abnormal, crashed, and errored.

**Comment by denrase** [comment]
@matejminar Thank you! I try out all calls before starting to implement and get back if i experience any issue. 🙇

**Comment by denrase** [comment]
@matejminar Hey there! Everything works as expected for now, just fetched the first apdex values on the client. Only thing that i'm missing is the `apdexThreshold` value. Where can i retrieve it from?

**Comment by matejminar** [comment]
Organization object should contain `apdexThreshold`.

This is how we calculate it on the web:
![image](https://user-images.githubusercontent.com/9060071/106490066-b72d7580-64b5-11eb-848f-439ebc080e70.png)

https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/projectDetail/projectScoreCards/projectApdexScoreCard.tsx

**Comment by denrase** [comment]
Ah yes, i see. It's not there when fetching the list of organizations, but it is when fetching individually.

**Comment by HazAT** [comment]
Closing this in favor of https://github.com/getsentry/sentry-mobile/issues/101
